### PR TITLE
Summation of Integrals #7827

### PR DESCRIPTION
**Pull request by Lakshya1605**
#7827

### CODE REVIEW

File: sympy/concrete/summations.py [hunk 1]
```diff
@@ -254,6 +254,28 @@ def _eval_derivative(self, x):
         else:
             return NotImplementedError('Lower and upper bound expected.')
 
+    def _eval_integral(self, x):
+        if isinstance(x, Symbol) and x not in self.free_symbols:
+            return x*self
```
Comment (asmeurer) [inline sympy/concrete/summations.py]: Wouldn't integrate already handle this case?

File: sympy/concrete/summations.py [hunk 2]
```diff
@@ -254,6 +254,28 @@ def _eval_derivative(self, x):
...
+
+        f, limits = self.function, list(self.limits)
+
+        limit = limits.pop(-1)
+
+        if limits:
```
Comment (asmeurer) [inline sympy/concrete/summations.py]: Is this logic tested?

File: sympy/concrete/summations.py [hunk 3]
```diff
@@ -254,6 +254,28 @@ def _eval_derivative(self, x):
...
+            f = self.func(f, *limits)
+
+        if len(limit) == 3:
+            _, a, b = limit
+            if x in a.free_symbols or x in b.free_symbols:
```
Comment (asmeurer) [inline sympy/concrete/summations.py]: Does this do the right thing if the same variable is used both as the summation index and a free variable (unfortunately, such things are allowed, c.f. `summation(x, (x, 1, x))`).

File: sympy/concrete/summations.py (hunk 2 above)
Comment (Lakshya1605) [inline sympy/concrete/summations.py]: Yes  this is tested ,actually limits returns an array containing tuples which in turn contain limits of summation "Some thing like this [(limits of the summation)]", so when we use limits.pop(-1) we get the tuple "(limits of summation)",Which is later used to extract each limits separately

File: sympy/concrete/summations.py (hunk 1 above)
Comment (Lakshya1605) [inline sympy/concrete/summations.py]: This case is just an extra method in the Class Sum just like "_eval_differentiation" which firsts differentiates the variable and then returns Sum class containing this new differentiated variable ,this method (_eval_integral) first integrates the variable and then return the Sum of integrated variable with the same limits of summation . I think integrate was not able to handle this case (that is first integrating and then summing)

File: sympy/concrete/summations.py (hunk 3 above)
Comment (Lakshya1605) [inline sympy/concrete/summations.py]: I don't think in integration and summation can be in cases like this ,and hence i have returned None on cases like this .

Comment (Lakshya1605) [inline sympy/concrete/summations.py]: ** I dont think integration and summation can be interchanged in cases like this and hence I have returned None in cases like this

File: sympy/concrete/summations.py (hunk 2 above)
Comment (asmeurer) [inline sympy/concrete/summations.py]: I mean in the test suite. Both of the tests you added have a single summation variable.

Comment (Lakshya1605) [inline sympy/concrete/summations.py]: @asmeurer  It has been test locally but i have not added any test in " test_integrals.p"y. Should i add  a test in "test_integral.py"?

Comment (moorepants) [inline sympy/concrete/summations.py]: Yes, please add any necessary tests.

**Comment by gxyd** [comment]
@Lakshya1605 can you please address the requested changes? Also resolve merge conflicts here.

**Comment by mightycandle** [comment]
Hello all. Is this issue open for changes?
Is there anyone working here?

**Comment by oscargus** [comment]
This seems to have been solved elsewhere in the code base, so closing this.

**Pull request closed without merging**